suite-common/message-system/config/config.v1.json

```diff
@@ -387,16 +296,16 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<23.10.1",
+                        "desktop": "<=24.5.4",
```

Comment:
24.5.4 is the Intermediary version signed by new certificate. This version looks to our servers for new updates, not Github. It is time to migrate majority of users to at least this version so we can kill Github releases in the long run eventually.